### DISCUSSION TITLE
Bearing Fit in Case

### DISCUSSION THREAD

**Discussion by marklogan17** [doudar/SmartSpin2k#211] [Q&A]
How tightly should the bearings fit in the case?  I finally got everything together and working but I'm getting some binding and it seems to be due to some play in the bigger gear under torque.  The bearings I have go in nicely but there's not enough friction to hold them in place if I flip the case upside down.  So I 'think' that as the knob is rotating and there's a bit of a twist in the device that there's enough movement in the gear position that its binding.  Could be another cause obviously, but that one seemed the mostly likely.

**Comment by marklogan17** [comment]
I put a tiny bit of electrical tape on the outside of the two bearings - not all the way around, just 1/2" long.  It served to increase the pressure fit and push the bearings a little closer to the stepper shaft.  My last test worked very well.  No binding at all.  Its also working from the QZ app although we have some tweaking to do for what is sent across as incline.  I'm currently printing out the 31T gear to see if that is better too.

**Comment by doudar** [comment]
Normally they’re tight enough that you really need to “friction fit” them in there either by using a little heat to soften the bearing recess of by gently sanding some material out.

**Reply by marklogan17** [reply]
That's what I was expecting, but in both my PLA print and my ABS print they weren't tight.  I'll put it down to my printer calibration or possibly the bearings I have are cheap and slightly undersized (left over from the figit spinner craze a few years ago when my son was printing them and selling them at school :) )

**Reply by doudar** [reply]
If you want to mess around with some more printing, try out the new version 2.0 case. I think it's really slick. If you don't have the PCB, the components might be a little tight but you may still get them to fit. The only different hardware you'll require is 4 `#8x2" Drywall screws. 
![image](https://user-images.githubusercontent.com/17362216/123875903-1629d600-d900-11eb-8103-10d7188744a4.png)
![image](https://user-images.githubusercontent.com/17362216/123876148-7a4c9a00-d900-11eb-9da6-e0d603719501.png)

**Reply by marklogan17** [reply]
Oh I like that.  I don't have the PCB so a little bit more difficult to use but I'm sure I could make it work.  I'm a little short on printing time as we're packing up to move so not sure if I'll get to trying this or not in the next few weeks.
